app/com/linkedin/drelephant/mapreduce/data/MapReduceApplicationData.java

```diff
@@ -148,6 +155,10 @@ public String getUsername() {
     return _username;
   }
 
+  public long getSubmitTime() {
```

Comment:
Can you add a doc for submitTime, startTime and finishTime saying what exactly the time refers to?

submitTime = Time when the job was submitted to the Resource Manager?